packages/logs/src/domain/trackConsoleLogs.ts

```diff
@@ -0,0 +1,49 @@
+import type { ClocksState } from '@datadog/browser-core'
+import { instrumentMethodAndCallOriginal, clocksNow, formatConsoleParameters } from '@datadog/browser-core'
+import type { StatusType } from './logger'
+
+export const ApiName = {
+  log: 'log',
+  debug: 'debug',
+  info: 'info',
+  warn: 'warn',
+} as const
+
+export const LogStatus = {
+  log: 'info',
+  debug: 'debug',
+  info: 'info',
+  warn: 'warn',
+} as const
+
+export type ApiNameType = typeof ApiName[keyof typeof ApiName]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏the `ApiName` variable does not seem to be used elsewhere
```suggestion
export type ApiNameType = "log" | "debug" | "info" | "warn"
```